x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/insight/index.tsx

```diff
@@ -240,19 +242,21 @@ const InsightComponent = ({
   relativeFrom,
   relativeTo,
 }: InsightComponentProps) => {
-  const isPlatinum = useLicense().isPlatinumPlus();
+  const insightsUpsellingMessage = useUpsellingMessage('investigation_guide');
```

Comment:
Nice change to take serverless into account. Consider renaming the variable:

```suggestion
  const investigationGuideUpsellingMessage = useUpsellingMessage('investigation_guide');
```

Reply:
I tried to keep it the same way it is used in other places in code [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/components/markdown_editor/editor.tsx#L76) and [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/index.ts#L26). Let me know if you think we should rename this variable in all places

Reply:
Yes, this is confusing because there's a feature key called `insights`, I thought this was named after it, but now I see it's named after the _insight_ uiPlugin. You are right, the name is perfectly correct, resolving.